ubidots/ubidots_in.js

```diff
@@ -90,6 +89,31 @@ module.exports = function (RED) {
         }
       });
     });
+
+    function defineOutputObject(topic, message) {
+      let finalObject = {};
+      let variable = topic;
+      if (useCustomTopics) {
+        variable = topic.substring(14);
+        if (topic.substring(topic.length - 3) === "/lv") {
+          finalObject[variable] = { value: JSON.parse(message.toString()) };
+        } else {
+          finalObject[variable] = JSON.parse(message.toString());
+        }
```

Comment:
I see that these lines are the same as the ones from +109 to +113. What do you think about creating a function so you don't need to duplicate them?